DiaHelperApplication/DiaHelperApplication/Controllers/ProfileViewController.swift

```diff
@@ -8,8 +8,298 @@
+        return textField
+    }()
+    
+    private var pumpIcon: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "pumpIcon")
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return imageView
+    }()
+    
+    private var pumpTextField: UITextField = {
+        let textField = UITextField()
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+    }()
+    
+    private var sensorIcon: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "sensorIcon")
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return imageView
+    }()
+    
+    private var sensorTextField: UITextField = {
+        let textField = UITextField()
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+    }()
+    
+    private var insulinIcon: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "insulinIcon")
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return imageView
+    }()
+    
+    private var insulinTypeTextField: UITextField = {
+        let textField = UITextField()
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+    }()
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        
         view.backgroundColor = .systemGray6
+        
+        if let user = UserManager.shared.getCurrentUser() {
+            currentUser = user
+        }
+        
+        setUpNameLabel()
+        setupEmailTextField()
+        setupNightscoutTextField()
+        setupBirthDateTextField()
+        setupYearOfDiagnosisTextField()
+        setupPumpTextField()
+        setupSensorTextField()
+        setupInsulinTypeTextField()
+        addSubviews()
+        addConstraints()
     }
+    
+    private func addSubviews() {
+        view.addSubview(profileImage)
+        view.addSubview(nameLabel)
+        view.addSubview(usernameLabel)
+        view.addSubview(emailIcon)
+        view.addSubview(emailTextField)
+        view.addSubview(nightscoutIcon)
+        view.addSubview(nightscoutTextField)
+        view.addSubview(birthDateIcon)
+        view.addSubview(birthDateTextField)
+        view.addSubview(yearOfDiagnosisIcon)
+        view.addSubview(yearOfDiagnosisTextField)
+        view.addSubview(pumpIcon)
+        view.addSubview(pumpTextField)
+        view.addSubview(sensorIcon)
+        view.addSubview(sensorTextField)
+        view.addSubview(insulinIcon)
+        view.addSubview(insulinTypeTextField)
+        
+    }
+    
+    private func addConstraints() {
+        view.addConstraints([
```

Comment:
It will be easier with a StackView as it will manage all constraints for the items inside